### PR TITLE
ci: Update existing bot comment

### PR DESCRIPTION
**Pull request by pawelpasterz**
`github-actions[bot]` comments are updated instead of creating a new one.
Used GMT+0 timezone for the timestamp

### CODE REVIEW

File: .github/workflows/ubuntu-workflow.yml [hunk 1]
```diff
@@ -36,10 +36,38 @@ jobs:
...
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          message: |
+          format: 'YYYY-MM-DD HH:mm:ss'
+          utcOffset: "+00:00"
+
+      - name: Find Comment
+        uses: peter-evans/find-comment@v1
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
```
Comment (piotradamczyk5) [inline .github/workflows/ubuntu-workflow.yml:50]: Probably for push events it will be null

File: .github/workflows/ubuntu-workflow.yml [hunk 2]
```diff
@@ -36,10 +36,38 @@ jobs:
         with:
           arguments: "clean build"
 
-      - name: Add ubuntu-workflow Build Scan URL to Pull Request
-        uses: mshick/add-pr-comment@v1
+      - name: Get current time
+        uses: 1466587594/get-current-time@v2
```
Comment (Sloox) [inline .github/workflows/ubuntu-workflow.yml:40]: what a strange usename

File: .github/workflows/ubuntu-workflow.yml (hunk 1 above)
Comment (Sloox) [reply .github/workflows/ubuntu-workflow.yml:50]: maybe skip past push events and only check for a creation of pull request? @pawelpasterz

**Comment by pawelpasterz** [comment]
@Sloox @piotradamczyk5 
Good catches, I changed logic a bit. Now workflow skips publishing url when event is different that 'pull_request'

**Review by piotradamczyk5 — APPROVED** [review]

**Review by jan-gogo — APPROVED** [review]

**Pull request merged**